pysteps/tracking/lucaskanade.py

```diff
@@ -0,0 +1,201 @@
+# -*- coding: utf-8 -*-
+"""
+pysteps.tracking.lucaskanade
+============================
+
+The Lucas-Kanade (LK) local feature tracking module.
+
+This module implements the interface to the local `Lucas-Kanade`_ routine
+available in OpenCV_.
+
+.. _OpenCV: https://opencv.org/
+
+.. _`Lucas-Kanade`:\
+    https://docs.opencv.org/3.4/dc/d6b/group__video__track.html#ga473e4b886d0bcc6b65831eb88ed93323
+
+.. autosummary::
+    :toctree: ../generated/
+
+    track_features
+"""
```

Comment:
Perhaps good to mention (for new users) here and/or in motion.lucaskanade.py that the same optical flow algorithm is used, but that the tracking functionality is defined in pysteps.tracking and vice versa for the standard motion derivation.

Reply:
Also, it can be briefly mentioned the differences between the two. To prevent any confusion.